app/styles/app/modules/icons.scss

```diff
@@ -87,7 +87,7 @@ svg.icon--m {
   margin-left: 0.2em;
   vertical-align: middle;
   border-radius: 50%;
-  border: 1px solid $pebble-grey;
+  border: 1px solid $cement-grey;
```

Comment:
Hmm.. I would not make this change global. It will affect all help icons across the app.